### ISSUE TITLE
[Bug Report][Windows][Android] AV1 video not being displayed

### ISSUE THREAD

**Issue by WaterBlueNewWorld** [media-kit/media-kit#351]
I have found an issue when trying to play an AV1 video after upgrading from: `media_kit 0.0.7+1` to `media_kit 1.1.3+1`.
Before this, my flutter app was able of playing an AV1 video that I had stored locally for testing purposes.
Here is a clip using `media_kit 0.0.7+1`:

https://github.com/media-kit/media-kit/assets/62195353/dac0662b-5b1d-436d-940b-80f4a63129b9

Once I updated to the mentioned version the subtitles are shown but not the video, also there is no error messages being thrown on the console.
Here is another clip this time using `media_kit 1.1.3+1`:

https://github.com/media-kit/media-kit/assets/62195353/dcc5d141-a9c3-492c-bf40-3504fb1668f9

Here is also the information given by ffprobe about the media file:
<details>
<summary>ffprobe output</summary>

```json
{
    "streams": [
        {
            "index": 0,
            "codec_name": "av1",
            "codec_long_name": "Alliance for Open Media AV1",
            "profile": "Main",
            "codec_type": "video",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "width": 1920,
            "height": 1080,
            "coded_width": 1920,
            "coded_height": 1080,
            "closed_captions": 0,
            "film_grain": 0,
            "has_b_frames": 0,
            "sample_aspect_ratio": "1:1",
            "display_aspect_ratio": "16:9",
            "pix_fmt": "yuv420p10le",
            "level": 8,
            "color_range": "tv",
            "refs": 1,
            "r_frame_rate": "24000/1001",
            "avg_frame_rate": "24000/1001",
            "time_base": "1/1000",
            "start_pts": 7,
            "start_time": "0.007000",
            "extradata_size": 17,
            "disposition": {
                "default": 1,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "DURATION": "00:24:06.744000000"
            }
        },
        {
            "index": 1,
            "codec_name": "opus",
            "codec_long_name": "Opus (Opus Interactive Audio Codec)",
            "codec_type": "audio",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "sample_fmt": "fltp",
            "sample_rate": "48000",
            "channels": 2,
            "channel_layout": "stereo",
            "bits_per_sample": 0,
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/1000",
            "start_pts": -7,
            "start_time": "-0.007000",
            "extradata_size": 19,
            "disposition": {
                "default": 1,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "ENCODER": "Lavc58.82.100 libopus",
                "DURATION": "00:24:06.788000000"
            }
        },
        {
            "index": 2,
            "codec_name": "ass",
            "codec_long_name": "ASS (Advanced SSA) subtitle",
            "codec_type": "subtitle",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/1000",
            "start_pts": -7,
            "start_time": "-0.007000",
            "duration_ts": 1446788,
            "duration": "1446.788000",
            "extradata_size": 5213,
            "disposition": {
                "default": 1,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "language": "eng",
                "DURATION": "00:24:05.677000000"
            }
        },
        {
            "index": 3,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 22528,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "Andes.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 4,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 150416,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "ANTQUABI.TTF",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 5,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 193732,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "apjfk.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 6,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 19204,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "Armorhide.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 7,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 71612,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "Asparagus Sprouts.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 8,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 28508,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "AVENIRLTSTD-HEAVY.OTF",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 9,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 23076,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "Baqa.otf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 10,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 42180,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "BetterHeather.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 11,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 47996,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "BlackChancery2.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 12,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 26392,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "CandelaBold.otf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 13,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 59284,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "CaviarDreams.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 14,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 580516,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "DK Crayon Crumble.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 15,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 106728,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "GandhiSans.otf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 16,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 114888,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "GandhiSansi.otf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 17,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 50160,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "HelveticaRdBd.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 18,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 177824,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "KGTenThousandReasonsAlt.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 19,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 40404,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "LIBERATIONSANS-BOLD.TTF",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 20,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 115712,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "MyriadPro-Semibold.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 21,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 119204,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "MyriadPro-SemiboldIt.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 22,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 119600,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "Terbium - Personal Use Only.ttf",
                "mimetype": "application/x-trutype-font"
            }
        },
        {
            "index": 23,
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": -630,
            "start_time": "-0.007000",
            "duration_ts": 130210920,
            "duration": "1446.788000",
            "extradata_size": 38152,
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "filename": "ufonts.com_centuryoldstyle-light.ttf",
                "mimetype": "application/x-trutype-font"
            }
        }
    ],
    "format": {
        "filename": "C:\\Users\\joel.avi├▒a\\Downloads\\revue.mkv",
        "nb_streams": 24,
        "nb_programs": 0,
        "format_name": "matroska,webm",
        "format_long_name": "Matroska / WebM",
        "start_time": "-0.007000",
        "duration": "1446.788000",
        "size": "273411915",
        "bit_rate": "1511828",
        "probe_score": 100,
        "tags": {
            "ENCODER": "Lavf58.42.101"
        }
    }
}
```
</details>

**Comment by alexmercerind** [comment]
Hi @WaterBlueNewWorld,

For now, you can pin your `media_kit_libs_windows_video` version.

In recent versions, we heavily modified the configuration to reduce the bundle size e.g.
9.0 MB -> 5.0 MB : Android ARM64
38.5 MB -> 26.1 MB : Windows x64

I did keep AV1. There's probably a mistake on our end.

**Comment by WaterBlueNewWorld** [comment]
It works now, many thanks for your response.

**Comment by deckerst** [comment]
I also noticed that AV1 playback started failing on Android when going from `media_kit_video` v1.1.2 to v1.1.3.

Some relevant logs when loading an AV1 video on a Samsung S10e (Android 12):
```
I/media_kit(13843): com.alexmercerind.media_kit_video.VideoOutputManager.create: 503194093376
I/media_kit(13843): com.alexmercerind.media_kit_video.VideoOutput: id = 0
I/media_kit(13843): com.alexmercerind.media_kit_video.VideoOutput: wid = 6634
D/media_kit(13843): MediaKitAndroidHelperOpenFileDescriptor: content://media/external/video/media/105239
D/media_kit(13843): get_env_result = -1
D/media_kit(13843): JavaVM::AttachCurrentThread Success
D/media_kit(13843): file_descriptor = 151
D/media_kit(13843): result = 126
I/flutter (13843): libmpv log: PlayerLog(prefix: ffmpeg/video, level: warn, text: av1_mediacodec: Unsupported or unknown profile)
I/flutter (13843): libmpv log: PlayerLog(prefix: ffmpeg/video, level: error, text: av1_mediacodec: MediaCodec 0x0 failed to start)
I/media_kit(13843): com.alexmercerind.media_kit_video.VideoOutputManager.setSurfaceTextureSize: 503194093376 960 540
I/flutter (13843): libmpv log: PlayerLog(prefix: vd, level: error, text: Could not open codec.)
I/flutter (13843): libmpv error: Could not open codec.
```

**Comment by alexmercerind** [comment]
On Windows, this was due to accidentally removed dav1d. This has been resolved in recent PR: #355.

**Comment by alexmercerind** [comment]
@deckerst,

I've made a commit in the same PR to resolve the issue on Android (you can try by changing one-liner locally):
https://github.com/media-kit/media-kit/pull/355/commits/c9094df6654b09247b76a1968ca526e0f21d6048

In the last update, [one change from upstream is that they try to open `hwdec`s sequentially (if one fails, try next, repeat)](https://github.com/mpv-player/mpv/pull/11874). This was important for Android, since it fixed issue with HEVC.
But, apparently, it broke AV1. For now, I removed AV1 from hardware decoding (AFAIK Android devices with AV1 hardware decoders are rare anyway), the rendering is still hardware accelerated. By this, good ol' libdav1d is utilized which is more reliable & most importantly, works.
